release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -3305,3 +3305,27 @@ link:https://access.redhat.com/solutions/6954669[{product-title} 4.8.39 containe
 ==== Updating
 
 To update an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-8-41"]
+=== RHSA-2022:2272 - {product-title} 4.8.41 bug fix and security update
+
+Issued: 2022-05-25
+
+{product-title} release 4.8.41, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:2272[RHSA-2022:2272] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:2270[RHBA-2022:2270] advisory.
```

Comment:
Both of these links are producing 404s for me. Please ensure that they are available prior to merging the PR.